lib/supabase/isPdfDownloaded.ts

```diff
@@ -0,0 +1,14 @@
+import { createClient } from './client'
+
+export const isPdfDownloaded = async (id: string) => {
```

Comment:
- `is` implies `read`
- `set` implies `update`
```suggestion
export const setPdfDownloaded = async (id: string) => {
```